libs/common/src/interfaces/response.interface.ts

```diff
@@ -5,3 +5,11 @@ export default interface IResponseType {
   data?: unknown;
   error?: unknown;
 };
+
+export default interface IResponse {
+  statusCode: number;
+  message?: string;
```

Comment:
As discussed with team, let's remove error and make message mandatory and keep other fields as it is.